libs/editor/build.gradle

```diff
@@ -74,10 +67,31 @@ dependencies {
             strictly wordPressAztecVersion
         }
     }
+    
+    // Forcing version due to https://github.com/facebook/SoLoader/issues/94
+    // To be removed with React Native 0.70+
+    implementation("com.facebook.soloader:soloader") {
+        version {
+            strictly '0.10.4'
+        }
+    }
+
+    implementation 'com.facebook.fresco:fresco:2.5.0'
+    implementation 'com.facebook.fresco:imagepipeline-okhttp3:2.5.0'
+    implementation 'com.facebook.fbjni:fbjni:0.2.2'
+    implementation 'com.facebook.react:hermes-engine:0.69.4:release@aar'
+    implementation 'com.facebook.react:react-native:0.69.4:release@aar'
+
     // This dependency will be substituted if the `local-builds.gradle` file contains
     // `localGutenbergMobilePath`. Details for this can be found in the `settings.gradle` file.
-    implementation "$rootProject.gradle.ext.gutenbergMobileBinaryPath:$rootProject.ext.gutenbergMobileVersion"
+    implementation ("$rootProject.gradle.ext.gutenbergMobileBinaryPath:$rootProject.ext.gutenbergMobileVersion") {
+        exclude module: 'hermes-engine'
+        exclude module: 'react-native'
+    }
 
+    // Required Aztec dependencies (they should be included but Jitpack seems to be stripping these out)
+    implementation "org.jsoup:jsoup:$jsoupVersion"
```

Comment:
I think we haven't updated the Aztec version, so I'm wondering why we need to add this dependency if we didn't have it before 🤔, what changed with the RN upgrade to producing this?

Reply:
This [comment](https://github.com/wordpress-mobile/WordPress-Android/pull/17066/files#r1042123549) is also related for this question.

Reply:
👋 @geriux @fluiddot !

I also stumbled upon this just now, that is, while working on the `org.jsoup:jsoup` dependency update (see [here](https://github.com/wordpress-mobile/WordPress-Android/issues/17569)). I just removed this dependency from the `libs:editor` module (it being unused and all) and the WPAndroid seems to be building/working just fine.

> I'm working on a draft PR as a follow-up to improve these, although to be able to force the release artifacts for all builds like (Jalapeno Debug) which are the builds PRs generate, we will have to keep a few implementations. 😅

Should I refrain from removing dependency and ignore this for now, that is, at least until this work above is done? Or maybe it is already done but I can't connect the dots just yet, not sure? 🤔

Thanking you in advance for your advise here! 🙏

Reply:
> and the WPAndroid seems to be building/working just fine.

Does the editor open correctly after this change?

Reply:
👋 @geriux and thanks for the reply!

> Does the editor open correctly after this change?

Reply:
FYI: I just pushed this [removal commit](https://github.com/wordpress-mobile/WordPress-Android/pull/18102/commits/e98d7e8f729084eabe8911943bb28796f3ac5c89), feel free to test [this PR](https://github.com/wordpress-mobile/WordPress-Android/pull/18102) yourself and let me know if everything works as you would have expected.

Reply:
Looks like the dependency `org.jsoup:jsoup` is used by Aztec-Android ([reference](https://github.com/wordpress-mobile/AztecEditor-Android/blob/23983f83a710ba5471c79416e6ce96febc63d6b9/aztec/build.gradle#L52)).

In fact, we set the version of that dependency [here in Gutenberg](https://github.com/WordPress/gutenberg/blob/b3a66dbc044d8c273744520161ea6436e07c1aaf/packages/react-native-aztec/android/build.gradle#L9). I'm not sure whether it can be removed or not, in any case, this could be tested by checking if the editor can be built and works as expected.